src/wwwroot/styles/views/_settings.scss

```diff
@@ -654,14 +654,13 @@
       li{
         border-left: 0px;
         border-right: 0px;
+        border-bottom: 0px;
+        padding: 0px;
```

Comment:
This is not looking good 🤕

Reply:
I will removed them in the next update 👍 thanks for your help Mauro